helm-charts/helm3/strimzi-kafka-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+Thank you for installing {{ .Chart.Name }}-{{ .Chart.Version }}
+
+To create a Kafka cluster refer to the following documentation.
+
+https://strimzi.io/docs/{{ .Chart.Version }}/#kafka-cluster-str
```

Comment:
I this is a broken link. I'm guessing the helm2 `NOTES.txt` will have the same broken link

Bonus points are available if there's some way to test this link so we catch regressions in the future.

Reply:
I updated it to https://strimzi.io/docs/operators/latest/using.html#deploying-cluster-operator-helm-chart-str since this is the only one I found close to helm-related docs. If the chart version has to be referenced somehow, I image a relevant check can be added to `check_docs.sh`. Please do let me know and I will see what I can do.